src/components/common/course-enrollments/data/__mocks__/programCourseEnrollments.json

```diff
@@ -1,80 +1,82 @@
-[
```

Comment:
The program course enrollments API has a slightly response than the enterprise course enrollments. For program course enrollments, the response is as follows: 

```
{
  "course_runs": [...]
}
```

whereas the enterprise course enrollments API response returns the array of course runs without the "course_runs" key. Updating the mock program course enrollments json file to include the "course_runs" key, matching the exact response.